contribs/drt/src/main/java/org/matsim/contrib/drt/analysis/DrtAnalysisControlerListener.java

```diff
@@ -214,6 +205,38 @@ public void notifyIterationEnds(IterationEndsEvent event) {
 				filename(event, "drt_boardings", ".csv"), filename(event, "drt_alightments", ".csv"), network);
 	}
 
+	private String getMinCountIdleVehiclesOverDayFormatted() {
+		double minCountIdleVehiclesOverDay = Double.MAX_VALUE;
+		double[] stayTaskProfile = vehicleOccupancyProfileCalculator.getNonPassengerServingTaskProfiles().get(new DrtTaskType(DrtTaskBaseType.STAY));
```

Comment:
you can also use `DrtStayTask.TYPE` instead of `new DrtTaskType(DrtTaskBaseType.STAY)`

Reply:
Thanks!